dask_expr/reductions.py

```diff
@@ -441,3 +442,26 @@ def aggregate_kwargs(self):
     def _simplify_up(self, parent):
         # We are already a Series
         return
+
+
+class MemoryUsage(Reduction):
+    _parameters = ["frame", "deep", "_index"]
+    _defaults = {"deep": False, "_index": True}
+    reduction_chunk = M.memory_usage
+    reduction_combine = staticmethod(
+        lambda x, is_dataframe: x.groupby(x.index).sum() if is_dataframe else x.sum()
+    )
```

Comment:
This seems complicated enough that maybe it deserves a normal `def ...` method

```python
@staticmethod
def reduction_combine(x, is_dataframe):
    ...
```

Reply:
Sounds good